build.sbt

```diff
@@ -1,22 +1,48 @@
-import Dependencies._
+import Dependencies.*
+
+// Your next release will be binary compatible with the previous one,
+// but it may not be source compatible (ie, it will be a minor release).
+ThisBuild / versionPolicyIntention := Compatibility.BinaryCompatible
+
+//TODO: after 3.1.0 release - clear up versionPolicyIgnored section
```

Comment:
let's ticket this TODO

Reply:
I will clean it up right after release, not sure it needs additional tickets.